Xamarin.Forms.Platform.UAP/FontImageSourceHandler.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Graphics.Canvas;
+using Microsoft.Graphics.Canvas.Text;
+using Microsoft.Graphics.Canvas.UI.Xaml;
+
+namespace Xamarin.Forms.Platform.UWP
+{
+	public sealed class FontImageSourceHandler : IImageSourceHandler
+	{
+		float _minimumDpi = 300;
+
+		public Task<Windows.UI.Xaml.Media.ImageSource> LoadImageAsync(ImageSource imagesource,
+			CancellationToken cancelationToken = default(CancellationToken))
+		{
+			if (!(imagesource is FontImageSource fontsource))
+				return null;
+
+			var device = CanvasDevice.GetSharedDevice();
+			var dpi = Math.Max(_minimumDpi, Windows.Graphics.Display.DisplayInformation.GetForCurrentView().LogicalDpi);
```

Comment:
Just a note about DPI on UWP: it can change during the lifetime of an app and even the control. The density can be changed via settings, but also by just dragging the app to a new monitor. I have a 200% surface and a 150% externbal display.

This might also change if accessibility is used - possibly on iOS and Android as well.

Reply:
Truth! But should fixing that use case block the PR? Kinda sounds like a corner case on a corner platform, no? Otherwise, we'll have to detect that the DPI changed and then run around and update glyphs? Sounds icky (or is it simple)? Do we think this is the only place we have that issue? Which is to say, if there are other places we cache DPI specific resources then how about we open an enhancement, subject it to prioritization, and fix 'em all at once.

I'm outta chaff.

Reply:
> we'll have to detect that the DPI changed and then run around and update glyphs?

it simple.

Reply:
I agree this is not a blocking issue at this time, but just wanted to make sure we all knew about this possibility. And, as you mention, we possibly have the same issue in other areas - maybe we need another issue...

Reply:
Nice. So we'll open a separate "Fix multimon DPI change on UAP" and fixing `FontImageSource` can be task one?

Reply:
Yeah. We probably can actually make this a public concept - the app may want to adjust to things like this. I was doing this for macOS for Xamarin.Essentials - a way to track when the resolution/scale changes. If the OS does not support it, then the event just never fires.

**macOS**
```csharp
NSNotificationCenter.DefaultCenter.AddObserver(NSApplication.DidChangeScreenParametersNotification, OnDidChangeScreenParameters);
```

**UWP:**
```csharp
var di = DisplayInformation.GetForCurrentView();
di.DpiChanged += OnDisplayInformationChanged;
```